lib/Doctrine/Persistence/ConnectionRegistry.php

```diff
@@ -26,14 +26,14 @@ public function getConnection(?string $name = null) : object;
     /**
      * Gets an array of all registered connections.
      *
-     * @return object[] An array of Connection instances.
+     * @return array<string, object> An array of Connection instances.
      */
     public function getConnections() : array;
 
     /**
      * Gets all connection names.
      *
-     * @return string[] An array of connection names.
+     * @return array<string, string> An array of connection names.
```

Comment:
Are these really indexed by name?

Reply:
Yes, I thought it was weird too but I didn't want to change the behavior in this PR.